Documentation~/en/install.md

```diff
@@ -0,0 +1,38 @@
+# Install package
+
+> [!NOTE]
+> Unity 2020.1 and 2019.4 are difference how to install the WebRTC package so please attention the Unity version you are using and follow the instruction below.
+
+## Case of Unity 2019.4
+
+1. Select `Window/Package Manager` in the menu bar.
+
+![Install Package Manager from menu bar](../images/install_select_packman_menu_unity2019.png)
+
+2. Check Package Manager window, Click `Advanced` button and enable `Show preview packages`.
+
+![Select show preview packages on advanced options](../images/install_select_show_preview_packages.png)
+
+3. Input `webrtc` to the search box at the top of the Package Manager window.
+
+![Search webrtc package](../images/install_search_webrtc_package.png)
+
+4. Click `Install` button at the bottom left of the window, and will start install the package.
+
+## Case of Unity 2020.1
+
+1. Select `Window/Package Manager` in the menu bar.
+
+![Install Package Manager from menu bar](../images/install_select_packman_menu_unity2020.png)
+
+2. Check Package Manager window, Click `+` button and select `Add package from git URL...`.
+
+![Select add package from git url](../images/install_select_add_package_from_git_url.png)
+
+4. Input the string below to the input field.
+
+```https://github.com/Unity-Technologies/com.unity.webrtc.git```
```

Comment:
Just to confirm, so you prefer to guide users to input URL instead of package name/version ?
Such as: com.unity.webrtc@0.x.x-preview

Reply:
OK, we seems to be able to use a tag to specify the version. https://docs.unity3d.com/Manual/upm-git.html

I will change it below
`https://github.com/Unity-Technologies/com.unity.webrtc.git#2.1.0-preview`

Reply:
I didn't mean tag though.
I meant, package name/version, like 
com.unity.webrtc@2.1.0-preview

instead of
https://github.com/Unity-Technologies/com.unity.webrtc.git#2.1.0-preview

But if you prefer to use URL, then it's all good for me.

Reply:
Oh, I got it. it is not git URL but works well!